test/functorch/test_control_flow.py

```diff
@@ -1422,11 +1422,21 @@ def wrapper(*args, **kwargs):
             return wrapper
 
         gm = make_fx(f_wrapper(map_fn))(torch.tensor(True), torch.ones([2, 3], requires_grad=False))
-        exp_graph = """\
-def forward(self, pred_1, x_1):
+
+        # The expected graph differs slight when running under Dynamo
+        if TEST_WITH_TORCHDYNAMO:
+            fwd_string = "def forward(self, arg0_1, arg1_1):"
+            impl_string = ("map_impl = torch.ops.map_impl(body_graph_0, 1, arg1_1, arg0_1);"
+                           "body_graph_0 = arg1_1 = arg0_1 = None")
+        else:
+            fwd_string = "def forward(self, pred_1, x_1):"
+            impl_string = ("map_impl = torch.ops.map_impl(body_graph_0, 1, x_1, pred_1);"
+                           "body_graph_0 = x_1 = pred_1 = None")
+
```

Comment:
This seems unexpected, do we know what's going on here? Dynamo is supposed to faithfully replicate what happens in eager mode

Reply:
That was a bug, latest commit fixes it. Thanks! (I initially thought it was ok since just the argument names changed)